### ISSUE TITLE
v0.0.0.65 listtransactions got the daemon killed by OOM-killer

### ISSUE THREAD

**Issue by walkjivefly** [Crowndev/crown#173]
Attempting to execute `crown-cli listtransactions "*" 99999999 0 false` consumed so much RAM (last observation was north of 6.7GB) that the OOM-killer terminated the process. This is ridiculous given the chain is only around 22k blocks and almost none of them have any actual txns other than  coinbase/coinstake.

**Comment by walkjivefly** [comment]
Same thing happens in v0.0.0.69

Note that it does not happen in v0.14.0.2. My collateral wallet there has 107156 txns in it and `listtransactions` completed in a few seconds and pushes up the crownd memory usage to 2.4GB

**Comment by walkjivefly** [comment]
Still happens in v0.18.71. Even asking for only 1000 transactions results in OOM death. This is moments before crown-qt got killed.
![Screenshot_2022-11-12_00-43-52](https://user-images.githubusercontent.com/17745270/201445129-37aaa058-b86c-471f-9d0f-177ea020e65f.png)

**Comment by BlockMechanic** [comment]
First, does normal listtransactions with wallet default limits work ok ? i think requesting crown-cli listtransactions "*" 99999999 0 false is excessive. It should work, but then if there are enough transactions to fill RAM....

**Comment by walkjivefly** [comment]
`listtransactions` works with the defaults and with arbitrary numbers in v0.0.0.75 *with a new wallet* (not the one the problem was originally encountered with).

I loaded that wallet into v0.0.0.7x a couple of days ago and reproduced the OOM. Probably it's related to whatever corruption caused the wallet to not show correct balances.

I can't actually run `listtransactions` on my current wallet for more than 448 txns because of the MRBLs induced 
```
mark@xub22:~/.crown/testnet3$ crown-cli listtransactions "*" 500
error code: -1
error message:
GetValueOutMap: value out of range
```
although `listunspent` does still work (well, not report an error).

Closing as probable one-off due to damaged wallet.
